### ISSUE TITLE
Create cohort-based standard table 1

### ISSUE THREAD

**Issue by schuemie** [OHDSI/FeatureExtraction#169]
The current [`createTable1()` function](http://ohdsi.github.io/FeatureExtraction/reference/createTable1.html) relies on concept-based covariates. We should create an alternative table 1 based on cohorts. For example, instead of 'Diabetes type 2' being based on the occurrence of a single condition concept, it should be based on a more complex definition, possibly involving antidiabetes treatments, lab values, etc.

I propose to use the same set of items as the current table 1 (see specs [here](https://github.com/OHDSI/FeatureExtraction/blob/main/inst/csv/Table1Specs.csv)), since those were selected before to be 'good enough'. For each item (except possible demographics) we should find a corresponding cohort definition. We could then
1. load those cohort definitions into FeatureExtraction. 
2. add a function in FeatureExtraction to retrieve all these cohort definitions (in a format suitable for CohortGenerator), 
3. add a function for creating cohort-based covariates specifically for this set of cohorts.
4. adapt the `createTable1()` function for working with this new table 1.

@gowthamrao: I'd imagine the cohort definitions would come from the OHDSI Phenotype Library.

Also looping in @pbr6cornell .

**Comment by schuemie** [comment]
I have extracted the concepts used in Table 1 [here](https://github.com/OHDSI/FeatureExtraction/blob/cohortBasedTable1/extras/table1Concept.csv), so people know which cohort definitions we need.

**Comment by gowthamrao** [comment]
Yes - here is a link to candidate cohorts

https://ohdsiorg.sharepoint.com/:x:/s/Workgroup-PhenotypeDevelopmentandEvaluation/EeNDdk7wsFxEtD1kxzH4Xf0BU4Ao2oNWOKb2-gL5tl1WPw?e=BQ6omz

**Comment by gowthamrao** [comment]
The version [3.1.1 of PhenotypeLibrary ](https://github.com/OHDSI/PhenotypeLibrary/releases/tag/v3.1.1)R package has the above referenced candidate cohorts.

**Comment by schuemie** [comment]
Hi @gowthamrao. The Excel sheet you referenced contains a lot of entries labeled 'not started' or status  = 'yes'. Would it be possible to create a clean table. where one column contains the cohort IDs as found the PhenotypeLibrary package?

**Comment by gowthamrao** [comment]
@schuemie  yes

Since building and maintaining these cohorts are now the responsibility of OHDSI Phenotype Development and Evaluation workgroup using the OHDSI Peer review process - i suggest we move this issue to https://github.com/OHDSI/PhenotypeLibrary

We can circle back to technical implementation once peer reviewed cohorts are available

Thoughts?

**Comment by gowthamrao** [comment]
FYI - Kevin Haynes and @gowthamrao are revisiting this effort now

**Comment by anthonysena** [comment]
I think we'll want to get #167 merged into FeatureExtraction and use that work to help drive the Table 1 functionality described in this issue.

I suppose once this is in place, we could potentially have a dependency on the PhenotypeLibrary (PL) package such that defining the Table 1 specs simply references the cohort(s) in the PL. Hoping this is what you had in mind @gowthamrao but let me know if I've misunderstood here.
